doc/api/http.md

```diff
@@ -542,6 +542,19 @@ changes:
 The `request.aborted` property will be `true` if the request has
 been aborted.
 
+### request.finished
+<!-- YAML
+added: v0.0.1
+-->
+
+* {boolean}
+
+The `request.finished` property will be `true` if [`request.end()`][]
+have been called.
+
+Note that `request.end()` will automatically be called if the request
```

Comment:
Can you drop "Note that" and make this sentence part of the same paragraph as the first sentence.

Reply:
Fixed